samples/Button.html

```diff
@@ -60,17 +60,20 @@
 		}
 	</style>
 </head>
-<body style="display: none; background-color:white;">
+<body style="display: none; background-color:white;" role="application" aria-labelledby="pageTitle">
 <div class="group">
 	<button is="d-button" label="Default button" onclick="alert(this.label + ' clicked.');"></button>
+	<button is="d-button" label="Button with alternative title" title="This is my alternative title" onclick="alert(this.label + ' clicked.');"></button>
+	<button is="d-button" onclick="alert(this.label + ' clicked.');">Button label by content</button>
 	<button is="d-button" class="d-button-primary" label="Blue button" onclick="alert(this.label + ' clicked.');"></button>
 	<button is="d-button" class="d-button-danger" label="Red button" onclick="alert(this.label + ' clicked.');"></button>
-	<button is="d-button" iconClass = 'd-icon-phone' label="Custom with icon" class="d-button-green" onclick="alert(this.label + ' clicked.');"></button>
-	<button is="d-button" iconClass = 'd-icon-phone d-big-icon' label="Custom with icon" class="d-button-green" onclick="alert(this.label + ' clicked.');"></button>
+	<button is="d-button" iconClass = 'd-icon-phone' label="Custom with icon and label" class="d-button-green" onclick="alert(this.label + ' clicked.');"></button>
+	<button is="d-button" iconClass = 'd-icon-phone d-big-icon' label="Custom with icon only" showLabel="false" class="d-button-green" onclick="alert(this.label + ' clicked.');"></button>
 </div>
 <div class="group">
-	Icon only button (screen readers should announce label as "Call"):
-	<button is="d-button" iconClass="d-icon-phone" title="Call"></button>
+	Icon only button (screen readers should announce label as "Button Call". Hovering with the mouse, a "Call" tooltip will be shown instead.):
+	<br>
+	<button is="d-button" iconClass="d-icon-phone" title="Call" label="Call Button"></button>
```

Comment:
This is labeled as "Icon only button" but it's still showing the button text.   (Conversely, on line 71 there's an icon only button.)